lib/baseTest.py

```diff
@@ -195,6 +196,34 @@ def load_configs(self):
                 config_value = json.load(fh)
 
             self.set_configs(config_variable_name, config_value)
+        self.extract_screen_settings_from_env()
+
+    # TODO: these screen settings should be moved to exec config in the future
+    def extract_screen_settings_from_env(self):
+        screen_settings = {
+            'browser-pos-x': self.env.DEFAULT_BROWSER_POS_X,
+            'browser-pos-y': self.env.DEFAULT_BROWSER_POS_Y,
+            'browser-width': self.env.DEFAULT_BROWSER_WIDTH,
+            'browser-height': self.env.DEFAULT_BROWSER_HEIGHT,
+            'viewport-width': self.env.DEFAULT_VIEWPORT_WIDTH,
+            'viewport-height': self.env.DEFAULT_VIEWPORT_HEIGHT,
+            'video-recording-pos-x': self.env.DEFAULT_VIDEO_RECORDING_POS_X,
+            'video-recording-pos-y': self.env.DEFAULT_VIDEO_RECORDING_POS_Y,
+            'video-recording-width': self.env.DEFAULT_VIDEO_RECORDING_WIDTH,
+            'video-recording-height': self.env.DEFAULT_VIDEO_RECORDING_HEIGHT
+        }
+        self.set_configs(self.config_name.EXEC, screen_settings)
+
+    def get_new_recording_size(self):
+        recording_width = min(self.exec_config['browser-width'] + 100, 1920)
+        if platform.system().lower() == "darwin":
```

Comment:
There is a `self.current_platform_name` for `platform.system()`, and it's lower case by default.